docker-compose.yml

```diff
@@ -54,55 +62,66 @@ services:
       CHARON_DATA_DIR: /charon/bootnode
       CHARON_P2P_BOOTNODES: ""
       CHARON_P2P_EXTERNAL_HOSTNAME: bootnode
-
-  vc0-lh:
+    volumes:
+      - ./:/charon/
+  #             _ _     _       _                 
+  # __   ____ _| (_) __| | __ _| |_ ___  _ __ ___ 
+  # \ \ / / _` | | |/ _` |/ _` | __/ _ \| '__/ __|
+  #  \ V / (_| | | | (_| | (_| | || (_) | |  \__ \
+  #   \_/ \__,_|_|_|\__,_|\__,_|\__\___/|_|  |___/
+  vc0:
     build: lighthouse
     networks: [cluster]
     depends_on: [node0]
+    env_file: .env
     environment:
       NODE: node0
     volumes:
-      - .:/charon
+      - ./lighthouse:/opt/charon/lighthouse
+      - .charon/cluster/node0:/opt/charon/keys
 
-  vc1-teku:
-    image: consensys/teku:latest
+  vc1:
+    image: consensys/teku:22.5.0
     networks: [cluster]
     depends_on: [node1]
+    restart: on-failure
     command: |
       validator-client
-      --network=auto
-      --beacon-node-api-endpoint="http://node1:16002" 
-      --validator-keys="/charon/node1:/charon/node1"
-      --validators-proposer-default-fee-recipient="0x0000000000000000000000000000000000000000"
+      --beacon-node-api-endpoint="http://node1:16002"
```

Comment:
we can also include this flag in teku-config.yaml and pass NODE aa env var

Reply:
Fair, hadn't considered that a yaml file would get env var substitution built into it. If it doesn't get it by default, I think it makes sense to have the common flags in the config file, and the flags that make each container different from one another in the compose file in one place for the dev to compare easily.